linter.py

```diff
@@ -17,33 +17,14 @@
 class Standard(NodeLinter):
     """Provides an interface to standard."""
 
-    syntax = ('javascript', 'html', 'javascriptnext', 'javascript 6to5', 'javascript (babel)')
     cmd = 'standard --stdin --verbose'
-    version_args = '--version'
-    version_re = r'(?P<version>\d+\.\d+\.\d+)'
-    version_requirement = '>= 3.7.2'
+    name = 'Standard'
     regex = r'^\s.+:(?P<line>\d+):(?P<col>\d+):(?P<message>.+)'
-    selectors = {
-        'html': 'source.js.embedded.html'
-    }
 
     html_pattern = re.compile(r'(^.*\n)\s+$', re.DOTALL)
 
-    npm_name = 'standard'
     defaults = {
         'enable_if_dependency': True,
-        'disable_if_not_dependency': False
+        'disable_if_not_dependency': False,
+        'selector': 'source.js'
     }
-
-    def run(self, cmd, code):
-        """
```

Comment:
I have to admit, that after reading the documentation I have no real idea of how to reproduce this behavior in V4 as `syntax` has been deprecated.